SampleApps/SPMTest/SPMTest.xcodeproj/project.pbxproj

```diff
@@ -368,22 +368,22 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/paypal/iOS-SDK.git";
 			requirement = {
-				branch = paypal_web_integration;
+				branch = "rename-card";
```

Comment:
Just want to make sure, do we have a ticket to point this back to `main` once all of the renaming work is completed? Want to make sure it doesn't get missed.

Reply:
@jaxdesmarais i dont have a ticket for it, but i was planning in fixing this later, thats why in the other module renaming i commented the GithubActions "SPMTest", it was creating all sort of problems with the merge, but ill write a ticket for it.

Reply:
@jaxdesmarais ticket here: https://paypal.atlassian.net/browse/DTNOR-810